author/concepts/what_is_metanorma.adoc

```diff
@@ -0,0 +1,23 @@
+== What is Metanorma?
+
+Metanorma is actually not one thing, but three:
+
+. The Metanorma toolchain: This is the software that you use to create documents and generate output. 
+. Document metamodels: Metanorma provides a document structure for standardization documents, specified in https://www.isotc154.org/projects/iso-36001/[ISO 36001]. Since each organization has different requirements for their standard documents, they use a https://www.metanorma.org/flavors/[“flavor”] of Metanorma. A flavor is document model that reflects the SDO’s needs by providing additional metadata. 
```

Comment:
```suggestion
. Document metamodels: Metanorma provides a document structure for standardization documents, specified in https://www.isotc154.org/projects/iso-36001/[ISO 36001]. Since each organization has different requirements for their standard documents, they use a https://www.metanorma.org/flavors/[“flavor”] of Metanorma. A flavor is a document model that reflects the SDO’s needs by providing additional metadata. 
```